INSTALL.md

```diff
@@ -134,4 +134,55 @@ target_link_libraries(foo PRIVATE ${OPENTELEMETRY_CPP_LIBRARIES})
 
 ## Build instructions using Bazel
 
-TBD
+### Prerequisites
+
+- A supported platform (e.g. Windows, macOS or Linux).
+Refer to [Platforms Supported](./README.md#supported-development-platforms)
+for more information.
+- A compatible C++ compiler supporting at least C++11.
+Major compilers are supported.
+Refer to [Supported Compilers](./README.md#supported-c-versions) for more information.
+- [Git](https://git-scm.com/) for fetching opentelemetry-cpp source code from repository.
+To install Git, consult the [Set up Git](https://help.github.com/articles/set-up-git/)
+guide on GitHub.
+- [Bazel](https://www.bazel.build/) for building opentelemetry-cpp API,
+SDK with their unittests. We use 3.7.2 in our build system.
+
+To install Bazel, consult the [Installing Bazel](https://docs.bazel.build/versions/3.7.0/install.html) guide.
+
+### Building as Standalone Bazel Project
+
+1. Getting the opentelementry-cpp source:
+
+   ```console
+   # Change to the directory where you want to create the code repository
+   $ cd ~
+   $ mkdir source && cd source
+   $ git clone https://github.com/open-telemetry/opentelemetry-cpp
+   Cloning into 'opentelemetry-cpp'...
+   ...
+   Resolving deltas: 100% (3225/3225), done.
+   $
+   ```
+
+2. Download the dependencies and build the source code:
+
+   ```console
+   $ bazel build //...
+   bazel build -- //... -//exporters/otlp/... -//exporters/prometheus/...
```

Comment:
Technically the exclusion here isn't right. It all builds in its entirety right now, including `otlp` - which is going to be the main thing. Helper script for windows:
https://github.com/open-telemetry/opentelemetry-cpp/blob/414a7314f7ab343533042f70eaea5810764fdd5d/tools/build-bazel.cmd#L4

Reply:
How about just invoke `tools/buildbazel.cmd` here which will keep both consistent?

Reply:
Maybe we need to add a script for Linux then, like `tools/build-bazel.sh` . But I actually like this document too.